README.md

```diff
@@ -105,4 +105,31 @@ E.g., `url`, `email`, etc.
 
 Doing this communicates semantics to user agents and assistive tech. For example, using `type="numeric"` might trigger the appropriate keyboard state on mobile.
 
+<a name="anchor-tag-usage"></a>
+### Anchor Tag Usage
+
+#### Placeholder Anchor Tags
+```
+<a>Text</a>
+```
+
+Placeholders are great for navigation or displaying hidden information on a page. Using a placeholder makes the element read as clickable to a screenreader so it is the preferred solution over a `<div>` or `<span>`.
+
+This element type first became valid in the HTML5 spec, so make sure your DOCTYPE isn't XHTML before use. If you're stuck in older HTML, use a link anchor tag.
+
+Reference: http://stackoverflow.com/questions/5292343/is-an-anchor-tag-without-the-href-attribute-safe
+
+#### Links
+```
+<a href="www.google.com">Search Google</a>
+```
+
+Use this style of anchor tag to create links to other pages.
+
+If using a link anchor tag as a placeholder (`<a href="#">Not a real link</a>), then be sure to add `ev.preventDefault()` to your JS click event. Otherwise, the page will jump to the top every time a user clicks your placeholder.
```

Comment:
I found that term used in multiple places on a variety of sites. Seems to be used to differentiate types of anchors since a "real" link uses the same tag. If we can agree that "anchor" means `<a>` and link refers to `<a href="">`, then I'll use that. Those terms didn't seem any more common than those that I used...and I learned anchor to synonymous with link at school, not that I had the best prof for web design.